Always gets the SHA1 hash from en-us, even if downloading other languages

Thanks so much for making this! For some reason, it always seems to get the SHA1 hash from the EN-US version, even if you're actually downloading another one.

See here:

Select a language for your Windows 11 ARM Professional ISO
Enter a Windows language tag (example: 'en-us'), or
     'p' to print the list of available languages, or
     'q' to exit the program without creating an ISO: de-de
Getting info on ESD from Microsoft
ESDURL : http://dl.delivery.mp.microsoft.com/filestreamingservice/files/4ba0241
7-fa20-4601-8e41-0472b5aed151/26100.1742.240909-0928.ge_release_svc_refresh_CLI
ENTCONSUMER_RET_A64FRE_en-us.esd
SHA1 : 4483ca4ad07f42a796d99d6c0820951a8494ec0a
SUCCESS
Downloading ESD from Microsoft                                                 Edition : Professional                                                         Language : de-de
Downloading Windows 11 Professional_de-de ESD file : 100.00%
ESD: /Users/pnorthup/Developer/24H2_esd2iso/26100.1742.240909-0928.ge_release_s
vc_refresh_CLIENTCONSUMER_RET_A64FRE_de-de.esd
SUCCESS
Validating download (this may take a minute or so)
[ERROR] ESD validation failed, error code 1
Expected hash = 4483ca4ad07f42a796d99d6c0820951a8494ec0a
        Calculated hash = 7db17a1adb2dc7aa16ad519967407da107a9583d
Program aborting

--- a/24H2_esd2iso
+++ b/24H2_esd2iso
@@ -200,7 +200,7 @@ downloadEsd() {
 	echo -e "Downloading ESD from Microsoft"
 	"${utilPath}" getesd --edition ${editionName} --lang ${esdLang} | tee ${wDir}/getesd.out
 	if grep -q  SUCCESS "${wDir}/getesd.out"; then
-		esdName=$(awk '/^ESD:/ {print $2}' "${wDir}"/getesd.out)
+		esdName=$(awk '/^ESD:/ {print $2}' "${wDir}/getesd.out")
 		esdName=$(basename "$esdName")
 		return 0
 	else

--- a/24H2_esd2iso
+++ b/24H2_esd2iso
@@ -16,7 +16,7 @@
 # 
 
 
-readonly versionID="v1.0.0 (20241004)"
+readonly versionID="v1.0.1 (20241007)"
 readonly version="24H2_esd2iso ${versionID}\n"
 readonly minDiskSpace=10
 
@@ -183,7 +183,7 @@ setupEsdDownload(){
 		esac
 	done
 echo -e "Getting info on ESD from Microsoft"
-	"${utilPath}" getesdurl -- edition "${editionName}" --lang ${esdLang} | tee "${wDir}/geturl.out"
+	"${utilPath}" getesdurl --edition "${editionName}" --lang ${esdLang} | tee "${wDir}/geturl.out"
 	if grep -q  SUCCESS "${wDir}/geturl.out"; then
 	    shaHash=$(awk '/^SHA1/ {print $3}' "${wDir}"/geturl.out)
 	    return 0

